examples/demo.html

```diff
@@ -148,22 +148,14 @@
             }
         })
         .geo()
-        .worldMap(map)
-        .marker([[-122.490402, 37.786453]], {
-            shape: 'triangle', fill: 'red',
-            cssClass: function (d, i, j) {
-                alert(d);
-            }
-         })
-        // .smallStatesCallouts()
-        // .WorldChoropleth({
-        //     topoJson: map,
-        //     cssClass: function (d) {
-        //         return  colorScale(d.properties.mapcolor7);
-        //     }
-
-        // })
+        // .worldMap(map)
+        .USMap(map)
+        .smallStatesCallouts()
+
         .tooltip()
+        // .marker([[-122.490402, 37.786453]], {
```

Comment:
why is this commented out?

Reply:
Removed changes to this file... is file is more a sandbox to play with the lib locally than anything that is published